docs/tutorials/notebooks/sparse_monge_displacements.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>The math doesn't completely render in the notebook.</p><p> </p><p>In general, this notebook lacks some subsections, would add some.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/374/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>where does the math not render here?</p>